### DISCUSSION TITLE
Open Tablet Driver filters?

### DISCUSSION THREAD

**Discussion by Sclafus** [ppy/osu#15306] [Ideas] [answered]
Since Open Tablet Driver is baked into Osu Lazer now, will it be possibile to add filters?
Usually filters are the driving factor to use a certain driver over another one for most osu players (eg [Devocub](https://github.com/Devocub/TabletDriver) vs [Hawku](https://github.com/hawku/TabletDriver)), it would be an amazing feature to have baked into osu!

**Comment by peppy** [comment, marked as answer]
If you can state which "filters" you need, we may consider adding them into the game. We probably won't support modular filters.

**Reply by Sclafus** [reply]
Smoothing filter and antichatter should be good enough for most people
